README.md

```diff
@@ -78,3 +79,52 @@ spec:
               name: https
 EOF
 ```
+
+## Tools
+
+### `app-config.yaml` template
+
+The `template_app_config_local.sh` is a tool to help in the generation of 
+ the `app-config.yaml` file. It contains several resources that can be either 
+ enabled or not depending on the provided information.
+
+The goal of the bash script is to transform the provided parameters into 
+ `jinja2` variables that will be replaced inside the script.
+
+Variables are to be provided as script options using the format 
+ `-variable value`.
+
+```bash
+./template_app_config_local.sh \
+  -SERVICE_ACCOUNT_TOKEN I_WONT_TELL \
+  -ARGOCD_ADMIN_PASSWORD password \
+  -ARGOCD_SERVER https://openshift-gitops-server-openshift-gitops.apps.qshift.snowdrop.dev/ \
+  -GITHUB_PERSONAL_ACCESS_TOKEN I_WONT_TELL_EITHER
+```
```

Comment:
The more parameters we will have the more difficult it will be to use this approach. If we follow the approach you propose, then we could certainly use the backstage app_config env approach for the same price (see: 
 https://backstage.io/docs/conf/writing/#environment-variable-overrides)

Personally, I would prefer that we propose an interactive mode for the user with default values (= when using the project locally). This is also not perfect as the list of the questions will increase over the time ...

Such an interactive mode could work like this:
? Backstage url [required] (localhost):
? Github Personal Access token [required]:

Reply:
Ok, I'll change the approach.

Reply:
Done.